mod/share.php

```diff
@@ -23,28 +23,29 @@ function share_init(App $a) {
 	} else {
 		$o = share_header($r[0]['author-name'], $r[0]['author-link'], $r[0]['author-avatar'], $r[0]['guid'], $r[0]['created'], $r[0]['plink']);
 
-		if ($r[0]['title'])
+		if ($r[0]['title']) {
 			$o .= '[b]'.$r[0]['title'].'[/b]'."\n";
+		}
```

Comment:
Can you please add a new line after this closing brace like you did with the `if () { ... }` blocks?